tooling.yml

```diff
@@ -0,0 +1,5 @@
+architecture-sniffer:
+  priority: 2
+
+code-sniffer:
+    level: 1
```

Comment:
Could we increase the level up 2?

Reply:
No, because it will automatically increase php-cs rules from Spryker to SprykerStrict. This module is not ready for SprykerStrict validations.